src/webui/www/public/newlabel.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en" dir="ltr">
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>QBT_TR(New Label)QBT_TR</title>
+    <link rel="stylesheet" href="css/style.css" type="text/css" />
+    <script type="text/javascript" src="scripts/mootools-1.2-core-yc.js" charset="utf-8"></script>
+    <script type="text/javascript" src="scripts/mootools-1.2-more.js" charset="utf-8"></script>
+    <script type="text/javascript">
+        window.addEvent('domready', function() {
+            $('newLabel').focus();
+            $('newLabelButton').addEvent('click', function(e) {
+                new Event(e).stop();
+                // check field
+                var labelName = $('newLabel').value;
```

Comment:
`var labelName = $('newLabel').value.trim()`

If there's a trailing space, two labels are created in the Web UI, but one in qBittorrent.
Also, we shouldn't allow leading/trailing white spaces since you can't create such labels from the regular UI. The server should refuse malformed requests or alter  them (e.g. by removing white spaces).